tests/adapters/compliance_tests/remove_interface_from_bond_test.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2015 Internap.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from hamcrest import assert_that, is_, none, empty, has_item, has_properties
+from netman.core.objects.exceptions import UnknownBond
+
+from tests.adapters.configured_test_case import ConfiguredTestCase
+
+
+class RemoveInterfaceFromBondTest(ConfiguredTestCase):
```

Comment:
Most of the comment for the add interface to bond should be addressed in this class too